COPYING

```diff
@@ -5,6 +5,6 @@ Redistribution and use in source and binary forms, with or without modification,
 
     * Redistributions of source code must retain the above copyright notice, this list of conditions and the following disclaimer.
     * Redistributions in binary form must reproduce the above copyright notice, this list of conditions and the following disclaimer in the documentation and/or other materials provided with the distribution.
-    * Neither the name of Redis nor the names of its contributors may be used to endorse or promote products derived from this software without specific prior written permission.
+    * Neither the name of Valkey nor the names of its contributors may be used to endorse or promote products derived from this software without specific prior written permission.
```

Comment:
@emaste like this is what you suggest? Fine by me.

```suggestion
    * Neither the name of the copyright holder nor the names of its contributors may be used to endorse or promote products derived from this software without specific prior written permission.
```

Reply:
I can't comment on whether it's legally permissible to make that (or any) change to the license text but yes that's what I mean.

I presume this sort of issue is precisely why the OSI version has it written as "copyright holder" (so that it doesn't need to be changed if the project forks, is renamed, etc.)

Reply:
I'll poll some other folks as well. This was one of the original files I left alone since I wasn't sure what to do about it.